roles/tpa_single_node/templates/manifests/guac/vexination-collector/Deployment.yaml.j2

```diff
@@ -35,47 +36,39 @@ spec:
             - collect
             - s3
             - --poll
-          env:
+          env: 
             - name: STORAGE_ACCESS_KEY
-              valueFrom:
-                secretKeyRef:
-                  key: root-user
-                  name: infrastructure-minio
+              value: {{ tpa_single_node_storage_access_key }}
             - name: STORAGE_SECRET_KEY
-              valueFrom:
-                secretKeyRef:
-                  key: root-password
-                  name: infrastructure-minio
+              value: {{ tpa_single_node_storage_secret_key }}
             - name: GUAC_S3_URL
-              value: http://infrastructure-minio:9000
+              value: {{ tpa_single_node_storage_endpoint }}
```

Comment:
This field is not mandatory for AWS S3 and can be wrapped with conditional statement like [here](https://github.com/trustification/trustification-ansible/blob/main/roles/tpa_single_node/templates/manifests/v11y/indexer/Deployment.yaml.j2#L100)